site-src/concepts/conformance.md

```diff
@@ -76,13 +76,26 @@ capabilities.
 
 By default, conformance tests will expect a `gateway-conformance` GatewayClass
 to be installed in the cluster and tests will be run against that. A different
-class can be specified with the `--gateway-class` flag along with the
-corresponding test command. For example:
+class can be specified with the `-gateway-class` flag along with the
+corresponding test command. For example, to run a conformance test against Istio,
+run:
 
 ```shell
-go test ./conformance --gateway-class my-class
+go test ./conformance/... -args -gateway-class=istio
 ```
 
+Other useful flags may be found in
+[conformance flags](https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/utils/flags/flags.go).
+For example, if you'd like to examine the objects in Kubernetes after your test runs, you can pass a flag to
+suppress cleanup:
+```shell
+go test ./conformance/... -args -gateway-class=istio -cleanup-base-resources=false
+```
+If you'd like to run a single test instead of the entire conformance suite, find your test name
+`(suite.ConformanceTest.ShortName)` and use it like this:
+```shell
+go test ./conformance/... --run TestConformance/YOURTESTNAME --gateway-class=istio
```

Comment:
We can use flag `supported-features` to specify the tests to run, for example:
```
go test ./conformance/... -supported-features=TLSRoute -gateway-class=my-class
```

Reply:
In this case, I wanted to explain how to run a single test case by name.  I haven't quite figured out how to use the supported-features flag to my advantage, so I would ask for someone else to document that.